webrender/src/resource_cache.rs

```diff
@@ -149,24 +149,23 @@ pub struct ResourceClassCache<K: Hash + Eq, V, U: Default> {
 
 pub fn intersect_for_tile(
     dirty: DeviceUintRect,
-    width: u32,
-    height: u32,
+    actual_tile_size: DeviceUintSize,
```

Comment:
perhaps, `clipped_tile_size`? We can even re-use the `ScreenRect` struct here

Reply:
`clipped_tile_size` sounds good to me. Which ScreenRect are you referring to?

Reply:
The only one we have - in `prim_store`:
```rust
#[derive(Debug, Copy, Clone)]
pub struct ScreenRect {
    pub clipped: DeviceIntRect,
    pub unclipped: DeviceIntRect,
}
```

Reply:
Oh, I doubt it'd make things any simpler in this case.

Reply:
yeah, I'm fine with just renaming the thing to `clipped_tile_size`

Reply:
I'll do it tomorrow, the build is also whining about a long line.